ci-operator/config/openshift-priv/cluster-baremetal-operator/openshift-priv-cluster-baremetal-operator-master.yaml

```diff
@@ -1,4 +1,8 @@
 base_images:
+  assisted-image-service:
```

Comment:
``priv`` configuration should be redundant (so you can either change it or not, it will have the same effect)
just FYI

Reply:
Yeah, I know, removed it.